root/etc/e-smith/templates/getmailrc/10retriever

```diff
@@ -4,8 +4,19 @@
     my $server = $account->prop('Server') || 'localhost';
     my $user = $account->prop('Username') || '';
     my $pass = $account->prop('Password') || '';
+    my $mailbox = $account->prop('Mailbox') || '';
+
     $OUT .= "type = $retr\n";
     $OUT .= "server = $server\n";
     $OUT .= "username = $user\n";
     $OUT .= "password = $pass\n";
+
+    if ((uc $mailbox) eq 'ALL' ) {
```

Comment:
We must disable the "Mailbox" prop effects if "Retriever" is not IMAP (POP3 does not support server-side folders)